app/src/main/kotlin/com/wire/android/ui/home/newconversation/common/SelectParticipantsButtonsRow.kt

```diff
@@ -99,8 +140,10 @@ private fun SelectParticipantsButtonsRow(
             verticalAlignment = Alignment.CenterVertically,
             modifier = modifier,
         ) {
+            val mainBtnText = if (showCount) "$mainButtonText ($count)" else mainButtonText
```

Comment:
mainBtnText :P mainButtonText would be better :P

Reply:
naming is hard :P